pkg/server/interfaces/api/application.go

```diff
@@ -433,6 +433,16 @@ func (c *application) GetWebServiceRoute() *restful.WebService {
 		Returns(400, "Bad Request", bcode.Bcode{}).
 		Writes(apis.ApplicationStatusResponse{}))
 
+	ws.Route(ws.GET("/{appName}/status").To(c.getApplicationStatusFromAllEnvs).
+		Doc("get application status from all envs").
+		Metadata(restfulspec.KeyOpenAPITags, tags).
+		Filter(c.RbacService.CheckPerm("envBinding", "detail")).
+		Filter(c.appCheckFilter).
+		Param(ws.PathParameter("appName", "identifier of the application ").DataType("string")).
+		Returns(200, "OK", []*apis.ApplicationStatusResponse{}).
```

Comment:
The response data is imperfection. Can't distinguish between different environments.

Reply:
This struct is as follows, it contain the the filed of  envName:

```json
type ApplicationStatusResponse struct {
	EnvName string            `json:"envName"`
	Status  *common.AppStatus `json:"status"`
}
```